konf/raw-site.yaml

```diff
@@ -0,0 +1,57 @@
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: ubuntu-com-security-api
+spec:
+  selector:
+    app: ubuntu-com-security-api
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+      targetPort: http
+
+---
+
+kind: Deployment
+apiVersion: extensions/v1beta1
```

Comment:
Since we updated K8s, we are now using `apiVersion: apps/v1`

Reply:
Fixed by using `konf production konf/site.yaml`.